### PR TITLE
Misc grammar changes to documentation.

### PR DESCRIPTION
**Pull request by Akkarinage**
Signed-off-by: Akkarinage <akkarin@rathena.org>

* **Addressed Issue(s)**: Grammar and some basic readability.

* **Server Mode**: Both

* **Description of Pull Request**: No issues to resolve, just making things make a bit more sense.

### CODE REVIEW

File: doc/sample/instancing.txt [hunk 1]
```diff
@@ -3,14 +3,14 @@
 //===== By: ==================================================
 //= Euphy
 //===== Last Updated: ========================================
-//= 20140129
+//= 20170303
 //===== Description: ========================================= 
-//= Contains elements of a basic instance script.
+//= Contains elements of a basic Single Char instance script.
```
Comment (Lemongrass3110) [inline doc/sample/instancing.txt]: single character?

File: doc/sample/instancing.txt [hunk 2]
```diff
@@ -36,14 +36,14 @@ prontera,151,190,6	script	Sample Instance	101,{
...
 		case 1:
-			.@create = instance_create(.@instance$);
+			.@create = instance_create(.@instance$, IM_CHAR);
 			if (.@create < 0) {
 				mes "[Sample Instance]";
 				switch (.@create) {
-					case -1: mes "ERROR: Invalid type."; break;
-					case -2: mes "ERROR: Party not found."; break;
+					case -1: mes "ERROR: Invalid type."; debugmes "Invalid instance type for ID " + instance_id(); break;
+					case -2: mes "ERROR: Char not found."; debugmes "Char/Party/Guild not found for ID " + instance_id(); break;
```
Comment (Lemongrass3110) [inline doc/sample/instancing.txt]: Message and debug message do not really fit?

File: doc/sample/instancing.txt [hunk 3]
```diff
@@ -36,14 +36,14 @@ prontera,151,190,6	script	Sample Instance	101,{
...
 					case -3: mes "ERROR: Instance already exists."; break;
-					case -4: mes "ERROR: No free instances."; break;
+					case -4: mes "ERROR: No free instances."; debugmes "No free instances. Check MAX_INSTANCE"; break;
```
Comment (Lemongrass3110) [inline doc/sample/instancing.txt]: Maybe add the current value like `(currently: "+MAX_INSTANCE+")`

File: doc/sample/instancing.txt [hunk 4]
```diff
@@ -66,8 +66,8 @@ prontera,151,190,6	script	Sample Instance	101,{
 	if (.@enter != 0) {
```
Comment (Lemongrass3110) [inline doc/sample/instancing.txt]: Use the IE_ constants here and in the cases.

File: doc/script_commands.txt [hunk 5]
```diff
@@ -107,15 +100,15 @@ Example:
...
 
 Upon loading all the files, the server will execute all the top-level commands 
 in them. No variables exist yet at this point, no commands can be called other 
-than those given in this section. These commands set up the basic server script 
-structure - create NPC objects, spawn monster objects, set map flags, etc. No 
-code is actually executed at this point except them. The top-level commands the 
-scripting are pretty confusing, since they aren't structured like you would 
-expect commands, command name first, but rather, normally start with a map name.
+than those given in this section. These commands set up the basic structure - create 
+NPC objects, spawn monster objects, set map flags, etc. No code is actually
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: Everything is an Object. Sounds similar...

File: doc/script_commands.txt [hunk 6]
```diff
@@ -247,8 +240,7 @@ degrees, where 0 means facing towards the top of the map. (So to turn the sprite
 towards the bottom of the map, you use facing 4, and to make it look southeast 
 it's facing 5.)
 
-Sprite id is the sprite number used to display this particular NPC. For a full 
-list of sprite id numbers see http://kalen.s79.xrea.com/npc/npce.shtml You may 
+Sprite id is the sprite number used to display this particular NPC. You may
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: Do we not support constants already too?

File: doc/script_commands.txt [hunk 7]
```diff
@@ -299,28 +291,25 @@ these floating NPC objects are for. More on that below.
...
 prices for items in different shops.
 
-Since trunk r12264 you can alternatively use "cashshop" in place of "shop"
-to use the Cash Shop interface, allowing you to buy items with special points
-(Currently stored as account variables in global_reg #CASHPOINTS and #KAFRAPOINTS.)
-This type of shop will not allow you to sell items at it, you may only
-purchase items here. The layout used to define sale items still count, and
+There are other types of shops available:
+cashshop - use "cashshop" in place of "shop" to use the Cash Shop interface, allowing 
+you to buy items with special points that are stored as account variables in `acc_reg_num`
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: Just drop the part where they are stored, because the table name is configurable.

File: doc/script_commands.txt [hunk 8]
```diff
@@ -299,28 +291,25 @@ these floating NPC objects are for. More on that below.
...
-item or special points from a variable. 'pointshop' only supported variable types
-are permanent character variables, temporary character variables, permanent
-local account variables, or permanent global account variables. These variables
-must be of integer type, not string. 'discount' flag is an option, makes the price
-at that shop is affected by discount skill.
+"itemshop" and "pointshop" use the Shop interface, allowing you to buy items with a specific
+item or special points from a variable. 'pointshop' only supports permanent character variables, 
+temporary character variables, permanent local account variables or permanent global account 
+variables. These variables must be of integer type, not string. 'discount' flag is an 
+option, makes the price at that shop is affected by discount skill.
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: Maybe it's just me, but "makes ... is affected ..." sounds really odd.

File: doc/script_commands.txt [hunk 9]
```diff
@@ -589,9 +578,7 @@ constants such as server defines and status options:
 Assigning variables
 --------- ---------
 
-As of rAthena revision 15982, variables can be accessed and assigned values directly
-without the use of the built-in 'set' function. This means that variables can be
-accessed and modified much like other programming languages.
+Variables can be accessed and modified much like other programming languages.
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: like in?

File: doc/script_commands.txt [hunk 10]
```diff
@@ -644,7 +631,7 @@ variable as if it was a normal variable:
 
     set @arrayofnumbers[0],1;
 
-You can also do sneaky things like using a variable (or an expression, or even a 
+You can also do things like using a variable (or an expression, or even a 
 value from an another array) to get at an array value:
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: an another

Comment (Lemongrass3110) [inline doc/script_commands.txt]: A few lines below there is another smiley you might wanna get rid of.

File: doc/script_commands.txt [hunk 11]
```diff
@@ -1327,7 +1311,6 @@ with other command, such as "if", but often used on its own.
 Label:
 	mes "This will be seen";
 
-Note by FlavioJS: goto's are "evil" and should be avoided if possible (ò_ó)
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: Maybe change this to something more general(just an example):
In programming goto has become a no-go and it is common practice not to use it. It should only be used if it really can't be avoided.

File: doc/script_commands.txt [hunk 12]
```diff
@@ -9433,5 +9416,5 @@ If <char id> is specified, the specified player is used rather than the attached
 
 ---------------------------------------
 
-Whew.
-That's about all of them.
+
+End of file.
```
Comment (Lemongrass3110) [inline doc/script_commands.txt]: just as worthless as the comment before

File: doc/skill_require_db.txt [hunk 13]
```diff
@@ -104,13 +104,13 @@ RequiredAmmoAmount: Amount of ammo needed to use the skill.
 
 RequiredState: The active 'State' needed to use the skill.
 
-none = Nothing special.
+none = Nothing.
```
Comment (Lemongrass3110) [inline doc/skill_require_db.txt]: no special state is required

File: doc/skill_require_db.txt [hunk 14]
```diff
@@ -104,13 +104,13 @@ RequiredAmmoAmount: Amount of ammo needed to use the skill.
...
 hidden = Requires hidden status by using Hiding, Cloaking, or Chasewalk.
 riding = Requires the player to ride either a Peco or a Dragon.
 falcon = Requires a Falcon.
 cart = Requires a Pushcart.
        For renewal, this state can be replaced by SC_PUSH_CART in 'RequiredStatuses' field.
-shield = Requires a shield equipped.
+shield = Requires an equipped shield.
 recover_weight_rate = Requires to be less than 50% weight.
```
Comment (Lemongrass3110) [inline doc/skill_require_db.txt]: to be at?

File: doc/skill_require_db.txt [hunk 15]
```diff
@@ -104,13 +104,13 @@ RequiredAmmoAmount: Amount of ammo needed to use the skill.
 
 RequiredState: The active 'State' needed to use the skill.
 
-none = Nothing special.
+none = Nothing.
 hidden = Requires hidden status by using Hiding, Cloaking, or Chasewalk.
 riding = Requires the player to ride either a Peco or a Dragon.
```
Comment (Lemongrass3110) [inline doc/skill_require_db.txt]: What about the cash mounts?

**Comment by Akkarinage** [comment]
Prof. Lemo, the new taskmaster 😝

File: doc/script_commands.txt (hunk 10 above)
Comment (Akkarinage) [inline doc/script_commands.txt]: I should probably look for my glasses. Well spotted!

File: doc/script_commands.txt [hunk 16]
```diff
@@ -247,9 +240,8 @@ degrees, where 0 means facing towards the top of the map. (So to turn the sprite
 towards the bottom of the map, you use facing 4, and to make it look southeast 
 it's facing 5.)
 
-Sprite id is the sprite number used to display this particular NPC. For a full 
-list of sprite id numbers see http://kalen.s79.xrea.com/npc/npce.shtml You may 
-also use a monster's ID number instead to display a monster sprite for this NPC. 
+Sprite id is the sprite number or constant used to display this particular NPC.
```
Comment (aleos89) [inline doc/script_commands.txt]: Sprite id -> Sprite ID

File: doc/script_commands.txt [hunk 17]
```diff
@@ -299,28 +291,25 @@ these floating NPC objects are for. More on that below.
 This will define a shop NPC, which, when triggered (which can only be done by 
 clicking) will cause a shop window to come up. No code whatsoever runs in shop 
 NPCs and you can't change the prices otherwise than by editing the script 
-itself. (No variables even exist at this point of scripting, so don't even 
-bother trying to use them.)
+itself.
 
 The item id is the number of item in the 'item_db.txt' database. If Price is set
```
Comment (aleos89) [inline doc/script_commands.txt]: item id -> item ID

File: doc/script_commands.txt [hunk 18]
```diff
@@ -299,28 +291,25 @@ these floating NPC objects are for. More on that below.
...
-item or special points from a variable. 'pointshop' only supported variable types
-are permanent character variables, temporary character variables, permanent
-local account variables, or permanent global account variables. These variables
-must be of integer type, not string. 'discount' flag is an option, makes the price
-at that shop is affected by discount skill.
+"itemshop" and "pointshop" use the Shop interface, allowing you to buy items with a specific
+item or special points from a variable. 'pointshop' only supports permanent character variables, 
+temporary character variables, permanent local account variables or permanent global account 
+variables. These variables must be of integer type, not string. 'discount' flag is an 
+option, makes the price at that shop become affected by discount skill.
```
Comment (aleos89) [inline doc/script_commands.txt]: option, makes -> optional value which makes

File: doc/script_commands.txt (hunk 11 above)
Comment (aleos89) [inline doc/script_commands.txt]: I agree with this.

File: doc/skill_require_db.txt (hunk 15 above)
Comment (aleos89) [inline doc/skill_require_db.txt]: This state only checks for Peco or Dragon specifically:
```c
		case ST_RIDING:
			if(!pc_isriding(sd) && !pc_isridingdragon(sd)) {
				clif_skill_fail(sd,skill_id,USESKILL_FAIL_LEVEL,0);
				return false;
			}
			break;
```
The cash shop mounts don't have any specific skill checks.

**Pull request merged**